src/features/analytics_panel/components/AnalyticsContainer/AnalyticsContainer.tsx

```diff
@@ -39,19 +39,24 @@ const AnalyticsContainer = () => {
         return <AnalyticsEmptyState />;
       }
       return (
-        <Tabs onTabChange={setTab} current={currentTab}>
-          <Tab name="INFO" id="data">
-            <AnalyticsDataList
-              data={dataList}
-              links={
-                (focusedGeometry?.source as any)?.meta?.externalUrls ??
-                undefined
-              }
-            />
-          </Tab>
-          {/*<Tab name="COMMUNITIES" id="communities">*/}
-          {/*  <AnalyticsCommunities />*/}
-          {/*</Tab>*/}
+        <Tabs>
+          <TabList style={{ display: 'none' }}>
```

Comment:
looks suspicious. What happens here?

Reply:
Yep. It's a temporary solutuin. Tabs component ist't really used in Analytics panel now since we only have one tab in it. So, for now I hided this single tab header with this code. Later on, once the AnalyticsCommunities tab will be activated, we can safely remove this hack and show full tab panel.